lab-matt/package.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
This is a good package.json in general, but has a few issues which I've noted below.